docs/devel/api_changes.md

```diff
@@ -456,50 +454,29 @@ regenerate auto-generated ones. To regenerate them:
    - run
 
 ```sh
-hack/update-generated-conversions.sh
+hack/update-codegen.sh
 ```
 
-If running the above script is impossible due to compile errors, the easiest
-workaround is to comment out the code causing errors and let the script to
-regenerate it. If the auto-generated conversion methods are not used by the
-manually-written ones, it's fine to just remove the whole file and let the
-generator to create it from scratch.
-
-Unsurprisingly, adding manually written conversion also requires you to add tests to
-`pkg/api/<version>/conversion_test.go`.
-
-## Edit deep copy files
-
-At this point you have both the versioned API changes and the internal
-structure changes done.  You now need to generate code to handle deep copy
-of your versioned api objects.
-
-The deep copy code resides with each versioned API:
+update-codegen will also generate code to handle deep copy of your versioned
+api objects. The deep copy code resides with each versioned API:
    - `pkg/api/<version>/deep_copy_generated.go` containing auto-generated copy functions
    - `pkg/apis/extensions/<version>/deep_copy_generated.go` containing auto-generated copy functions
 
-To regenerate them:
-   - run
-
-```sh
-hack/update-generated-deep-copies.sh
-```
-
-## Edit json (un)marshaling code
-
-We are auto-generating code for marshaling and unmarshaling json representation
+It also auto-generating code for marshaling and unmarshaling json representation
```

Comment:
This may be misleading, but it's a different script - it:
hack/update-codecgen.sh
vs
hack/update-codegen.sh  [without "c"]